commit b174a0df054929efd7804922750c71adeb8e0b57
Author: Peter Stephenson <p.w.stephenson@ntlworld.com>
Date:   Fri Apr 12 22:47:22 2013 +0100

    31261: updates needed with latest texinfo

diff --git a/Doc/Zsh/compsys.yo b/Doc/Zsh/compsys.yo
index 2991c1f4a..4a436dae5 100644
--- a/Doc/Zsh/compsys.yo
+++ b/Doc/Zsh/compsys.yo
@@ -4414,8 +4414,7 @@ tt(ambiguous), tt(special-dirs), tt(list-suffixes) and tt(file-sort)
 described above.
 )
 findex(_pick_variant)
-xitem(tt(_pick_variant) [ tt(-b) var(builtin-label) ] [ tt(-c)
-var(command) ] [ tt(-r) var(name) ])
+xitem(tt(_pick_variant) [ tt(-b) var(builtin-label) ] [ tt(-c) var(command) ] [ tt(-r) var(name) ])
 item(   var(label)tt(=)var(pattern) ... var(label) [ var(args) ... ])(
 This function is used to resolve situations where a single command name
 requires more than one type of handling, either because it
diff --git a/Doc/ztexi.yo b/Doc/ztexi.yo
index ed0236bde..513452b14 100644
--- a/Doc/ztexi.yo
+++ b/Doc/ztexi.yo
@@ -193,8 +193,8 @@ def(startlist)(0)(\
   startitem()\
 )
 def(endlist)(0)(\
-  NOTRANS(@item)+NL()\
-  enditem()\
+  enditem()+NL()\
+  NOTRANS(@sp 1)\
 )
 def(list)(1)(\
   item(ARG1)()\
